src/pages/LoginHandler/index.js

```diff
@@ -1,146 +1,11 @@
-    dispatch(routerRedux.push(`/private`));
-  };
-
-  enableSubmitBtn = () => {
-    this.setState({
-      variantVal: 'primary',
-      btnColor: 'white',
-      btnDisabled: 'false',
-    });
-  };
-
-  disableSubmitBtn = () => {
-    this.setState({
-      variantVal: 'tertiary',
-      btnColor: 'black',
-      btnDisabled: 'true',
-    });
-  };
-
-  handleUserNameInputChange = username => {
-    const { password } = this.state;
-    this.setState({
-      username,
-    });
-    if (password !== '' && username !== '') {
-      this.enableSubmitBtn();
-    } else {
-      this.disableSubmitBtn();
-    }
-  };
-
-  handlePassWordInputChange = password => {
-    const { username } = this.state;
-    this.setState({
-      password,
-    });
-    if (username !== '' && password !== '') {
-      this.enableSubmitBtn();
-    } else {
-      this.disableSubmitBtn();
-    }
-  };
-
-  handleLoginSubmit = () => {
-    const { username, password } = this.state;
-    if (username === 'admin' && password === 'admin') {
-      this.setLoggedIn({ username });
-    } else {
-      alert('Wrong username/password pair');
-    }
-  };
-
-  render() {
-    const { variantVal, btnColor, btnDisabled } = this.state;
-    const form = (
-      <div className={styles.section}>
-        <Form>
-          <FormGroup label="Email address" isRequired fieldId="horizontal-form-name">
-            <TextInput
-              isRequired
-              type="text"
-              id="horizontal-form-name"
-              aria-describedby="horizontal-form-name-helper"
-              name="horizontal-form-name"
-              onChange={this.handleUserNameInputChange}
-            />
-          </FormGroup>
-          <FormGroup label="Password" isRequired fieldId="horizontal-form-password">
-            <TextInput
-              isRequired
-              type="password"
-              id="horizontal-form-password"
-              name="horizontal-form-password"
-              onChange={this.handlePassWordInputChange}
-            />
-          </FormGroup>
-          <FormGroup fieldId="remember-me">
-            <Checkbox
-              label="Keep me logged in"
-              id="alt-form-checkbox-1"
-              name="alt-form-checkbox-1"
-              className={styles.check}
-            />
-          </FormGroup>
-          <ActionGroup>
-            <Button
-              isBlock
-              variant={variantVal}
-              onClick={() => this.handleLoginSubmit()}
-              className={styles.btn}
-              id="submitBtn"
-              {...(btnDisabled === 'true' ? { isDisabled: 'true' } : {})}
-            >
-              <Title headingLevel="h4" size="xl" style={{ color: btnColor }}>
-                Submit
-              </Title>
-            </Button>
-          </ActionGroup>
-        </Form>
-      </div>
-    );
-    return <AuthLayout toPreview={form} heading="Log into your Pbench Acount" backOpt="true" />;
-  }
+function LoginHandler() {
```

Comment:
This page component isn't being referenced in the codebase. Do we need this component definition?

Reply:
@gurbirkalsi LoginHandler is the login page that we have in the dashboard now. It is referenced when the user hits `/login` URL.